docs/books/computer-games/creepy-computer-games/computer_nightmare/computer_nightmare.py

```diff
@@ -0,0 +1,55 @@
+''' You are a late-night computer addict and you have
+fallen asleep at the keyboard. Suddenly your computer
+comes alive and start hurling numbers and abuse at you.
+To beat it you have to match the numbers as they appear
+on the screen. Your starting score of 300 is increased
+if you hit the right number and decreased if you don't.
+If you can get your score up to 500 the computer will
+give up and you win, but if it goes down 0, you'll become
+a slave of your computer. It's a real nightmare! can you stay sane? '''
```

Comment:
This "flavour text" should be in the index.rst file, so it's part of the main document.